actionpack/lib/action_dispatch/journey/formatter.rb

```diff
@@ -15,7 +15,7 @@ def initialize(routes)
 
       def generate(name, options, path_parameters, parameterize = nil)
         constraints = path_parameters.merge(options)
-        missing_keys = nil # need for variable scope
+        missing_keys = nil # Needed for variable scope.
```

Comment:
I don't think this comment is needed - it's obvious to an experienced rubyist that we're initialising the variable so that it's correctly scoped. The Rails codebase isn't meant to be a tutorial on Ruby.